fe/fe-core/src/main/java/com/starrocks/catalog/mv/MVTimelinessRangePartitionArbiter.java

```diff
@@ -177,6 +179,16 @@ protected MvUpdateInfo getMVTimelinessUpdateInfoInLoose() {
             String mvPartitionName = addEntry.getKey();
             mvUpdateInfo.addMvToRefreshPartitionNames(mvPartitionName);
         }
+        Set<Table> refBaseTables = mv.getRefBaseTablePartitionColumns().keySet();
+        boolean allOlapTables = refBaseTables.stream().allMatch((Predicate<Table>) t -> t instanceof OlapTable);
+        if (allOlapTables) {
```

Comment:
1. Why only check when `allOlapTables` is true?
2. How about list partitioned mvs in `MVTimelinessListPartitionArbiter` ?

Reply:
1. The `version` of mv based on jdbc tables is never updated, thus `hasStorageData()` is always false. I'm not sure if there is other type of `Table` that will not update `version`, so only `OlapTable` is allowed here. 
2. Not tested.

Reply:
done modification to list partitioned mv